src/components/select-field/SelectField.scss

```diff
@@ -29,3 +29,17 @@
     max-height: 340px; // show at most 12 items in the dropdown menu
     overflow-y: auto;
 }
+
+.bdl-SelectField {
+    .bdl-SelectFieldDropdown {
```

Comment:
Is this nesting needed? It'll increase specificity and shouldn't be required since we're using SUIT.